tests/testthat/test-edge.R

```diff
@@ -0,0 +1,13 @@
+library(testthat)
+library(gfpop)
+context("edge")
+
+test_that("edge is vectorized", {
+  expect_silent({
```

Comment:
hi @vrunge I was expecting to be able to creates null edges via this vectorized syntax, is that OK? if so please fix the warnings that appear. if not please change to expect_error